packages/redux-dynostore-react-redux-subspace/test/subspaced.test.js

```diff
@@ -7,12 +7,25 @@
  */
 
 import React from 'react'
+import { createStore } from 'redux'
 import configureStore from 'redux-mock-store'
 import { Provider, connect } from 'react-redux'
 import { SubspaceProvider } from 'react-redux-subspace'
 import { mount } from 'enzyme'
 
+import dynostore from '../../redux-dynostore-core/lib/index'
```

Comment:
Was about to merge this then noticed this in my final scan.  This is a problematic import for 2 reasons:
1. it is pathing outside of the package into another - this is brittle if the other package ever gets refactored
2. it's referencing the bundled output, which means the other package would need to be built before this test would work - it's purely coincidental that that occurs as part of bootstrapping lerna for us.

in this case, importing it from the core package as you would any other npm dependency is all that is needed.  lerna takes care of the rest for us:

```js
import dynostore from '@redux-dynostore/core'
```